build.gradle

```diff
@@ -82,39 +49,27 @@ sourceSets {
     }
     test {
         java {
-            srcDirs = ["src/test"]
+            srcDirs = ["src/test", "src/CommunityCommons/javasource"]

```

Comment:
Have you successfully run unit tests from `src/CommunityCommons/javasource` with the Gradle `test` task? I'm pretty sure the Java-based unit tests there are just two examples from the unittesting module, and they would need a runtime because the `Core` API is accessed.